README.md

```diff
@@ -26,7 +26,7 @@
   <a href="https://www.npmjs.com/packages/chai">
     <img
       alt="node:?"
-      src="https://img.shields.io/badge/node-%3E=0.10-blue.svg?style=flat-square"
+      src="https://img.shields.io/badge/node-%3E=>4.0-blue.svg?style=flat-square"
```

Comment:
Hello @andersonaguiar, thank you so much for the PR

I think that you need to remove the `>` that comes before `4.0`

We want the badge to look like this
![](https://img.shields.io/badge/node-%3E=4.0-blue.svg?style=flat-square)
Instead of
![](https://img.shields.io/badge/node-%3E=%3E4.0-blue.svg?style=flat-square)

The link should be:
[https://img.shields.io/badge/node-%3E=4.0-blue.svg?style=flat-square](https://img.shields.io/badge/node-%3E=4.0-blue.svg?style=flat-square)

@chaijs/chai  am I missing something?